src/main/kotlin/io/polygon/kotlin/sdk/Version.kt

```diff
@@ -1,6 +1,6 @@
 package io.polygon.kotlin.sdk
 
 object Version {
-    const val name = "v4.2.1"
+    const val name = "v4.3.0"
```

Comment:
Specific call out here that I'm bumping minor version since there is a new feature with no breaking changes. Let me know if v4.2.2` would be more appropriate.